package.json

```diff
@@ -13,16 +13,16 @@
     "styled-components": "^3.2.5"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build && styleguidist build",
+    "start": "node scripts/setup.js && react-scripts start",
```

Comment:
I'd extract to a separate script just in case we decide to modify it. Something like `"check": "node scripts/setup.js"` and then I'd use `pre` prefix everywhere it should be used.

Reply:
What I don't like about that, it clutters output even more:

```
> component-driven-development@0.1.0 exercise /Users/tema/_/egghead/component-driven-development
> npm run setup && styleguidist server --exercise "1"


> component-driven-development@0.1.0 setup /Users/tema/_/egghead/component-driven-development
> node scripts/setup.js
```

Reply:
Okay your call